src/app/pages/dashboard/dashboard.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="px-0 container-fluid">
-  <app-http-table></app-http-table>
+  <app-demo-table></app-demo-table>
```

Comment:
I think `demo` isn't ideal as a naming convention for a component, but for now we can leave it.